app/javascript/client/home/utils.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+
+interface size {
+    width: number,
+    height: number
+}
+
+export const useWindowSize = ()=>{
```

Comment:
Where's the `: FC<{}>`? :joy: Mostly kidding, just calling out your deep devotion to TypeScript